style.css

```diff
@@ -13,6 +13,7 @@
     position: absolute;
     top: 50%;
     left: 50%;
+    gap: 2px;
```

Comment:
Add this at the top most style of the CSS class to put the whole form at the center.
```css
    position: absolute;
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
```